data/curi-adi.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "Aditya Shrivastava",
+  "type": "personal",
+  "bio": "IT professional",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AadiDreamer23",
+      "icon": "FaTwitter"
+    }
```

Please add the missing bracket here! `]`

```suggestion
    }
  ]
```